front/public/locales/en/operationalStudies/project.json

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
Thanks for that, it will be useful ! Any way you could add the corresponding and missing e2e test for the multiple selection and deletion of projects ?